src/theme/interaction.rs

```diff
@@ -57,15 +62,55 @@ fn apply_interaction_palette(
     }
 }
 
+#[derive(Resource, Asset, Reflect, Clone)]
+pub struct ButtonAssets {
```

Comment:
I think the naming should be consistent between this and `trigger_interaction_sfx`. If not, a doc comment should be added to this resource that makes the connection clear, like `/// Interaction sound effects for buttons.`.

As things currently stand, `trigger_interaction_sfx` only requires the `Interaction` component, so it can apply to things that may not be considered buttons.

Reply:
Agreed. I'm fine with using interaction or button everywhere, just need to be consistent

Reply:
Makes sense! I'll align the name.